packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -490,6 +493,12 @@ export class GarbageCollector implements IGarbageCollector {
             }
             return dataStoreGCDetailsMap;
         });
+
+        // Initialize the base state. The base GC data is used to detect and log when inactive / deleted objects are
+        // used in the container.
+        this.initializeBaseStateP.catch((error) => {
+            this.mc.logger.sendErrorEvent({ eventName: "FailedToInitializeGC" }, error);
```

Comment:
Failures in this.initializeBaseStateP would result in every summary to fail and be logged there as well.
So I wonder
- do we can about log it here as well?
- should summaries succeed (assume everything is connected) even if GC process fails?
-

Reply:
Summaries should not succeed if GC failed. Otherwise, the summary will have incorrect GC data which is not acceptable.
Updated this to throw an error.